light-client-js/Cargo.toml

```diff
@@ -23,7 +23,7 @@ default = ["console_error_panic_hook"]
 serde = { version = "1.0", default-features = false, features = [ "derive" ] }
 serde_json = { version = "1.0", default-features = false }
 # TODO(thane): Remove once https://github.com/rustwasm/wasm-bindgen/issues/2508 is resolved
-syn = { version = "=1.0.65", default-features = false }
+syn = { version = "1.0.95", default-features = false }
```

Comment:
If we're unpinning the version here (which seems justified) we don't need to explicitly import `syn`.

```suggestion
```